src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            required
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            min="1" max="100"
```

Comment:
Each attribute should be on separate line (in case if they are not written in one line)